browser/brave_profile_prefs.cc

```diff
@@ -256,6 +262,11 @@ void RegisterProfilePrefs(user_prefs::PrefRegistrySyncable* registry) {
   registry->RegisterStringPref(kGeminiRefreshToken, "");
 #endif
 
+  // Bitcoin.com widget
+#if BUILDFLAG(BITCOIN_DOT_COM_ENABLED)
+  registry->RegisterBooleanPref(kMoonpayHasBoughtBitcoinDotComCrypto, false);
```

Comment:
Move the things inside this block to `Something::RegisterPrefs(registry);`, see `speedreader` in this file.

Reply:
done